### DISCUSSION TITLE
Next steps for Decap CMS

### DISCUSSION THREAD

**Discussion by mjohenneken** [decaporg/decap-cms#6704] [Q&A] [answered]
As announced on the [Netlify Blog](https://www.netlify.com/blog/netlify-cms-to-become-decap-cms/) there will be some changes to the company structure and a change of ownership of the git repositories. Netlify will become Decap CMS.

It seems that activity from Netlify in this repository stopped in May 2022. I would like to know what the plans are for continuing the maintenance of this repository and future development.

**Comment by danurbanowicz** [comment]
Also interested to know how they're going to tackle that beast of an issues backlog. I think they have already started triaging through them, which is positive.

Decap CMS posted a rough outline of their plans [here](https://decapcms.org/blog/2023/02/introducing-decap).

**Comment by martinjagodic** [comment, marked as answer]
This is the plan:

1. Bring the project back from the dead. Update dependencies, and make sure that all test pass.
2. Go through open issues and see what we solved along the way.
3. Review open pull requests and merge what can be merged

When along those points we will release the first Decap version is yet to be determined. 

We ask for your patience as we solve deprecation issues. We are making progress on our fork but it's not possible yet to determine a time frame.

**Reply by nosovk** [reply]
it could be good idea to switch from react to something else. Have a look at https://github.com/sveltia/sveltia-cms it works and seems to bee much easier to support

**Reply by mmkal** [reply]
That seems extreme. A smaller change that would make it similarly more maintainable though could be incrementally adopting react-query to replace the redux-style reducers everywhere. My team have some custom GitHub api calls for a few add-on features like batch changes for multiple entries pushed to a PR branch, and we just wrap the GitHub api in react-query - there's virtually no state management or boilerplate, and it doesn't conflict with the existing implementation.

**Reply by eaton** [reply]
Updating the rich text editor — which has fallen quite a few releases behind — would also be a fabulous step; there are a number of outstanding behavioral bugs in current browsers that trace back to this IIRC

**Reply by martinjagodic** [reply]
Updating the markdown editor is at the core of update efforts.

**Reply by ngdangtu-vn** [reply]
Hello, could I ask for reviewing my PR please? (https://github.com/decaporg/decap-cms/pull/6836)

**Comment by bberenberg** [comment]
Is there a timeline for the dependency cleanup portion? Specifically https://github.com/decaporg/decap-cms/issues/6499

**Reply by martinjagodic** [reply]
We are getting closer every day, but it's not possible yet to estimate when exactly this will be done.

**Comment by Migacz85** [comment]
Hi guys I'm just curious how the progress is going with this project?

**Reply by martinjagodic** [reply]
This is the progress so far: https://github.com/decaporg/decap-cms/pull/6852

**Comment by almereyda** [comment]
It's been released as version 3 today:

- https://github.com/decaporg/decap-cms/discussions/6878

**Reply by m3dwards** [reply]
Congrats on getting the release out. I've been waiting for this!
